housekeeper/store/api/crud/read.py

```diff
@@ -25,15 +25,31 @@
 from housekeeper.store.models import Bundle, File, Tag, Version
 from housekeeper.store.filters.tag_filters import TagFilter, apply_tag_filter
 
-from .base import BaseHandler
+from housekeeper.store.api.base import BaseHandler
 
 LOG = logging.getLogger(__name__)
 
 
-class FindHandler(BaseHandler):
+class ReadHandler(BaseHandler):
     """Handler for searching the database"""
 
-    def bundles(self):
+    @staticmethod
+    def _get_query(table: Type[ModelBase]) -> Query:
+        """Return a query for the given table."""
+        return table.query
+
+    def _get_join_version_bundle_query(self) -> Query:
+        """Return version bundle query."""
+        return self.Version.query.join(Version.bundle)
+
+    def _get_join_file_tag_query(self) -> Query:
+        """Return file tag query."""
+        return self.File.query.join(File.tags)
+
+    def _get_join_version_query(self, query: Query):
+        return query.join(Version)
+
+    def get_bundles(self):
```

Comment:
Make sure to update the use of any renamed function in the hk module in the cg repository as well. It should be limited to a single file where any name changes need to be made since the hk module in cg acts as a wrapper around the housekeeper package.

Reply:
Alright.. that I will need to see for then! thanks for the heads up